easybuild/easyconfigs/p/pyBigWig/pyBigWig-0.3.13-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'PythonPackage'
+
+name = 'pyBigWig'
+version = '0.3.13'
+versionsuffix = '-Python-%(pyver)s'
+
+github_account = 'deeptools'
+homepage = 'https://github.com/%(github_account)s/%(name)s'
+description = """A python extension, written in C, for quick access to bigBed files and access to and creation of
+ bigWig files."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+checksums = ['1e0114bb106b7b05613d8be753759b578e7a16b3cd03f21bedcdace13b2bea08']
+
+dependencies = [('Python', '3.6.6')]
+
+use_pip = True
+
+download_dep_fail = True
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = ['python lib/python%(pyshortver)s/site-packages/pyBigWigTest/test.py']
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
```

Comment:
@SethosII This line is not needed, it's taken care of automatically with `PythonPackage`